src/gui/guiScrollBar.h

```diff
@@ -17,6 +17,12 @@ the arrow buttons where there is insufficient space.
 using namespace irr;
 using namespace gui;
 
+enum ScrollBarArrowVisibility {
+	HIDE,
+	SHOW,
+	DEFAULT
+};
```

Comment:
You can put that enum into the `GUIScrollBar` class:
```cpp
class GUIScrollBar : public IGUIElement
{
public:
	enum ArrowVisibility {
		HIDE,
		SHOW,
		DEFAULT
	};
//...
};
```
Outside of this class you can access it with `GUIScrollBar::ArrowVisibility` and its values eg. with `GUIScrollBar::HIDE`.
The forward declaration (`guiFormSpecMenu.h`, line 40)  might be possible with `enum GUIScrollBar::ArrowVisibility;` or with
```cpp
enum GUIScrollBar::ArrowVisibility {
	HIDE,
	SHOW,
	DEFAULT
};
```
(or just include the header in the header).